docs/usage/8.md

```diff
@@ -0,0 +1,298 @@
+# Restricted Network Image Mirroring for OpenShift 4.2
+
+## Introduction
+This guide was develeped using the [generic instructions](https://docs.openshift.com/container-platform/4.2/operators/olm-restricted-networks.html) for mirroring operators from OLM. 
```

Comment:
Insufficient because it took 2 days for me to cobble together a reproducable set of steps for ours and OCS operators... they're just a guide using etcd as an example. If you want to mirror ours this is a concise set of steps...